src/main/java/core/basesyntax/service/operationwithdata/AddOperation.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.operationwithdata;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.dto.FruitDto;
+import core.basesyntax.fruitmodel.Fruit;
+import java.util.Optional;
+
+public class AddOperation implements FruitOperationService {
+    @Override
+    public int apply(FruitDto fruitDto) {
+        Fruit fruit = new Fruit(fruitDto.getFruitName());
+        FruitDao fruitDao = new FruitDaoImpl();
```

Comment:
make FuitDao class field instead of local, we don't need to create a separate instance of it per each method call

Reply:
done